### ISSUE TITLE
[Q] Any plans to move to @playwright/test?

### ISSUE THREAD

**Issue by jeremyzilar** [storybookjs/test-runner#78]
I noticed that `jest-playwright` package recommends using `@playwright/test` over `playwright`. Any plans to move this this in the future, since it has typescript support out of the box? 

- `jest-playwright` https://github.com/playwright-community/jest-playwright#configuration
- `@playwright/test` https://playwright.dev/docs/intro#installation

**Comment by yannbf** [comment]
Hey @jeremyzila, we use jest-playwright as the playwright runner does not support file transformations and we need that in order to turn stories into tests. If they ever support that we could totally support it as a runner too.

I wonder what's your use case, specifically the typescript part. Our runner also has support for typescript!

**Comment by horacioh** [comment]
what I'm trying to do is to use Vitest instead of Jest to run the test-runner. this will minimize the amount of setup, configs and time to run tests I assume by a substantial amount.

can storybook rely on Vite to transform the stories instead? that would be neat!

**Comment by yannbf** [comment]
Hey @patak-dev do you think we can already experiment with vite? Will it be able to accept transformations like jest?

**Comment by patak-dev** [comment]
Yes, from the code you shown me last time, I dont see any blockers

**Comment by yannbf** [comment]
Alright, I'll do some experimentations with it! Thanks!

**Comment by andykenward** [comment]
I think we might be able to use `@playwright/test` as `playwright` just appears to be a wrapper to `playwright-core`. Which is used in `@playwright/test` aswell.

**Comment by andykenward** [comment]
It looks like `jest-playwright` is installing `playwright-core` too https://github.com/playwright-community/jest-playwright/blob/95b83774be71e3859875ccdcc27e0e5aef28c3ff/package.json#L49.

**Comment by andykenward** [comment]
> It looks like `jest-playwright` is installing `playwright-core` too https://github.com/playwright-community/jest-playwright/blob/95b83774be71e3859875ccdcc27e0e5aef28c3ff/package.json#L49.

Ah nope. Still need to install `playwright` to use `jest-playwright-preset`. Would've thought that `playwright` would be a [`peerDependencies`](https://github.com/playwright-community/jest-playwright/blob/95b83774be71e3859875ccdcc27e0e5aef28c3ff/package.json#L53-L58) for `jest-playwright-preset`.

**Comment by bryanjtc** [comment]
Any progress on this? @yannbf Did running with vitest instead of jest worked?

**Comment by resource11** [comment]
Oooo, I'd love to know this as well, @yannbf. Thanks for all your work on Storybook!

**Comment by rc-glean** [comment]
+1, @yannbf. Thank you so much for all your work on storybook!

Just to offer another datapoint, here's (IMO) one compelling reason to support `@playwright/test`. It has access to the experimental ssim-cie94 image comparator (cross-link https://github.com/microsoft/playwright/issues/13873#issuecomment-1344821266).

Context: a common problem is developers might generate screenshots locally on an arm mac, and execute CI tests against x86 linux.

This yields non-determinism. From https://gitnet.fr/deblan/playwright/commit/6d82460a02dec3393490bcab7b619f668f03e6c6

> As of Nov 30, 2022, we identified the following sources of
> non-determenistic rendering for Chromium:
> - Anti-aliasing for certain shapes might be different due to the
>   way skia rasterizes certain shapes.
> - Color blending might be different on `x86` and `aarch64`
> architectures.

As it stands, non SaaS (chromatic/percy) solutions like Amex's https://github.com/americanexpress/jest-image-snapshot/issues/326 does not have access to this experimental comparator even when setting package.json resolutions
